cmd/ufs/ufs.go

```diff
@@ -30,11 +49,14 @@ func main() {
 
 		_, err = ufs.NewUFS(func(s *stub.Server) error {
 			s.FromNet, s.ToNet = c, c
-			s.Trace = nil  // log.Printf
+			s.Trace = nil // log.Printf
+			if *cpuprofile != "" {
+				s.Trace = profile
```

Comment:
now that is some wild stuff.

But I think its' kind of abusing the intent of s.Trace, which is about printing trace messages. Can we do this differently?